classes/class-loader.php

```diff
@@ -87,39 +52,34 @@ public static function get_instance( $services, $view_class ) : Loader {
 	 * Checks requirements and if its ok it hooks the hook_plugin method on the 'init' action which fires
 	 * after WordPress has finished loading but before any headers are sent.
 	 * Most of WP is loaded at this stage (but not all) and the user is authenticated.
-	 *
-	 * @param array  $services The Controller services to inject.
-	 * @param string $view_class The View class name.
 	 */
-	private function __construct( $services, $view_class ) {
+	private function __construct() {
 
-		$this->load_files();
-		$this->load_services( $services, $view_class );
+		$this->setup_autoload();
+		$this->load_services();
 		$this->load_commands();
 		$this->check_requirements();
 
 		// Load Blocks.
-		$this->blocks = [
-			new Blocks\Accordion(),
-			new Blocks\Articles(),
-			new Blocks\CarouselHeader(),
-			new Blocks\Columns(),
-			new Blocks\Cookies(),
-			new Blocks\Counter(),
-			new Blocks\Covers(),
-			new Blocks\Gallery(),
-			new Blocks\Happypoint(),
-			new Blocks\Media(),
-			new Blocks\SocialMedia(),
-			new Blocks\SplitTwoColumns(),
-			new Blocks\Spreadsheet(),
-			new Blocks\SubMenu(),
-			new Blocks\SubPages(),
-			new Blocks\TakeActionBoxout(),
-			new Blocks\Timeline(),
-			new Blocks\SocialMediaCards(),
-			new Blocks\ENForm(),
-		];
+		new Blocks\Accordion();
```

Comment:
No point keeping these instances in memory if they're never used, which was the case for `$this->blocks`